R/theme_e61.R

```diff
@@ -42,15 +40,16 @@
 #'
 
 theme_e61 <- function(y_top = TRUE,
-                      adj = -12,
                       fix_left = 0,
+                      adj = 0,
                       legend = c("none", "bottom", "top", "left", "right"),
                       legend_title = FALSE,
+                      keep_legend = FALSE,
```

Comment:
This is new? Please document it in the roxygen

Reply:
Yep it's a hacky way of not deleting the legend formatting when updating the chart. I'll add to the documentation.